versions.gradle

```diff
@@ -7,7 +7,7 @@
 
 ext {
     versionCode = 62
-    versionName = '4.1.1'
+    versionName = '4.1.1-SNAPSHOT'
```

Comment:
How it will be removed for public release?

Reply:
manually, we discussed it internal